docs/postingest.md

```diff
@@ -0,0 +1,174 @@
+# Postingest
```

Comment:
In the picture, I think, we need a condition block from "resender" saying "Is the message stale?" and yes to have arrows to the queues

Reply:
This is more of a network diagram showing the links between the components and what you're describing is in the internal logic of the resender. There is some text in here which describes the resender process.